src/Models/PurchaseOrderContent.php

```diff
@@ -60,7 +60,14 @@ public function vendor()
 
     public function item()
     {
-        return $this->vendorItem->item;
+        return $this->hasOneThrough(
+            config('mojito.itemClass'),
+            config('mojito.vendorItemClass', VendorItemPivot::class),
+            'id',
+            'id',
+            'item_vendor_id',
+            'item_id'
+        )->withTrashed();
```

Comment:
El $this->vendorItem anterior fa un ->withTrashed().
No tinc clar que amb el hasOneThrough tingui sentit posar-ho

Reply:
@aleixgil això se m'escapa, aquí no et puc ajudar, caldria fer tests per assegurar-ho. Podries afegir els named parameters i així quedaria més clar què és cada cosa.

Reply:
@alexrevo named parameters afegits